scss/6-components/advisernet/_related-content.scss

```diff
@@ -66,3 +66,11 @@
     }
   }
 }
+
+// :-( this whole file is a bit much
```

Comment:
Yeah, fine for this to be a bit gnarly for now as we're going to pull this out anyway